api/Cargo.toml

```diff
@@ -9,6 +9,7 @@ description = "Makes a kernel compatible with the bootloader crate"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+log = "0.4.17"
```

Comment:
The `log` crate defines some global items, which might lead to problems when the user's kernel also depends on a different version of the `log` crate. Since we only need the `LevelFilter` enum I think it's better to define our own copy of that enum and avoid this dependency.